### ISSUE TITLE
Dark theme / CSS refactor

### ISSUE THREAD

**Issue by heyitsmeuralex** [decent-chat/decent#185]

**Comment by heyitsmeuralex** [comment]
Note #187 removes the `:host` class prefix stuff in favour of static classnames (e.g. `._xyz123` -> `.channel-list`), making theming a lot more viable.

**Comment by towerofnix** [comment]
Generally, I think it would make sense to refactor the CSS to use more descriptive variable names, e.g. "pane background". For example:

```css
body {
  --gray-900: #fff;
  --pane-background: var(--gray-900);
}

.sidebar (or :host, whatever) {
  background: var(--pane-background);
}
```

**Comment by PullJosh** [comment]
While we're already looking at refactoring CSS, would there be any benefit to switching to [BEM](http://getbem.com/introduction/)?

**Comment by heyitsmeuralex** [comment]
CSS has now been refactored. Only thing left is to make a dark theme or theming UI in the client.

**Comment by towerofnix** [comment]
Implemented in #250.
